### DISCUSSION TITLE
Dropdown Caret?

### DISCUSSION THREAD

**Discussion by jasper502** [orchidjs/tom-select#384] [Q&A] [answered]
I am using tom-select in my Railes app with tailwind css.  None of my selects have the caret so I added the CSS from the default in chrome but it still does not display:

```
.ts-wrapper {

  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='none' viewBox='0 0 20 20'%3e%3cpath stroke='%236b7280' stroke-linecap='round' stroke-linejoin='round' stroke-width='1.5' d='M6 8l4 4 4-4'/%3e%3c/svg%3e");  
  background-position: right 0.5rem center;
  background-repeat: no-repeat;
  background-size: 1.5em 1.5em;

}
```

Tom-select on the left - stock browser select on the right:

![image](https://user-images.githubusercontent.com/1505829/169177434-5ffb8e38-7c87-4314-8113-91c267fd904c.png)

When I refresh the page I can see the caret for a split second - it seems to be there just hidden  - any tips here to fix this?

**Comment by jasper502** [comment, marked as answer]
lol discovered by accident:

```
.ts-control {

  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='none' viewBox='0 0 20 20'%3e%3cpath stroke='%236b7280' stroke-linecap='round' stroke-linejoin='round' stroke-width='1.5' d='M6 8l4 4 4-4'/%3e%3c/svg%3e");
  background-position: right 0.5rem center;
  background-repeat: no-repeat;
  background-size: 1.5em 1.5em;

}
```

**Comment by mcmaddox** [comment]
Thank you! Just added Tom-Select to my app & this was stumping me...
